asset/gulpfile.js

```diff
@@ -40,39 +37,65 @@ var gulp = require('gulp'),
         'yaml'
     ];
 
+// Images
 gulp.task('images', function () {
     return gulp.src('img/*')
         .pipe(gulp.dest('../theme/img/'));
 });
 
+// Icons
+gulp.task('icons', function () {
+    return gulp.src('node_modules/font-awesome/fonts/**.*')
+        .pipe(gulp.dest('../theme/css/fonts'));
+});
+
+// JavaScript
 gulp.task('scripts', function () {
     var prismComponents = [];
     for (var component in prism) {
-        prismComponents[component] = 'node_modules/prismjs/components/prism-' + prism[component] + '.min.js';
+        prismComponents[component] = 'node_modules/prismjs/components/prism-' + prism[component] + '.js';
     }
 
-    return gulp.src(prismComponents.concat([
-            'node_modules/prismjs/plugins/normalize-whitespace/prism-normalize-whitespace.min.js',
-            'node_modules/anchor-js/anchor.min.js',
-            'js/base.js',
-            'js/scripts.js'
-        ]))
-        .pipe(concat({path: 'scripts.js'}))
-        .pipe(uglify({mangle: false}))
+    return gulp.src(
+        prismComponents.concat(
+            [
+                'node_modules/prismjs/plugins/normalize-whitespace/prism-normalize-whitespace.min.js',
+                'node_modules/jquery/dist/jquery.min.js',
+                'node_modules/bootstrap-sass/assets/javascripts/bootstrap.min.js',
+                'node_modules/anchor-js/anchor.min.js',
+                'js/base.js',
+                'js/component-list.js'
+            ]
+        )
+    )
+        .pipe(concat({path : 'scripts.js'}))
+        .pipe(uglify({mangle : false}))
         .pipe(rev())
         .pipe(gulp.dest('../theme/js/'))
-        .pipe(rev.manifest('../assets.yml', {merge: true}))
+        .pipe(rev.manifest('../assets.yml', {merge : true}))
```

Comment:
Not sure if space before `:` in js objects is common, usually I can see without space before and one space after.